AppController/djinn.rb

```diff
@@ -4774,6 +4774,9 @@ def prep_app_dashboard()
       setup_app_dir(AppDashboard::APP_NAME, true)
     }
 
+    # Allow fewer dashboard instances for small deployments.
+    min_dashboards = [3, get_all_appengine_nodes.length].min
```

Comment:
The number of appengine nodes being the determining factor doesn't make much sense to me since we wouldn't be specifically putting on dashboard on each node.

@obino any thoughts?